ML/MLClassification.py

```diff
@@ -272,33 +297,47 @@ def main():
         inputCfg = yaml.load(ymlCfgFile, yaml.FullLoader)
     print('Loading analysis configuration: Done!')
 
-    print('Loading data files: ...', end='\r')
-    PromptDf = LoadDfFromRootOrParquet(inputCfg['input']['prompt'])
-    FDDf = LoadDfFromRootOrParquet(inputCfg['input']['FD'])
-    DataDf = LoadDfFromRootOrParquet(inputCfg['input']['data'])
-    print('Loading data files: Done!')
+    print('Loading and preparing data files: ...', end='\r')
+    PromptHandler = TreeHandler(inputCfg['input']['prompt'])
```

Comment:
Does the `TreeHandler` support also a list of files as input?

Reply:
Not at the moment. We could add this possibility in hipe4ml

Reply:
Yes indeed I agree! The feature would be useful to be able to pass more than one file (i.e. to use prompt+FD as signal or merge different pT bins). Let me know if I can help you with hipe4ml!